extensions-rhel-9.6.yaml

```diff
@@ -0,0 +1,90 @@
+# RPMs as operating system extensions, distinct from the base ostree commit/image
+# https://github.com/openshift/enhancements/blob/master/enhancements/rhcos/extensions.md
+# and https://github.com/coreos/fedora-coreos-tracker/issues/401
+
+extensions:
+  # https://issues.redhat.com/browse/RFE-4177
+  # Not available in c9s
```

Comment:
I'm not sure why this `Not available in c9s` comment is here. Can you add more context?

Reply:
The `crun-wasm` package is not available in c9s repos. It's needed for OCP extensions so we pull it from RHEL 9.4 repos

Reply:
I just removed the comment and added another one below with more clarification